apps/web-giddh/src/app/daybook/daybook.component.html

```diff
@@ -119,7 +119,7 @@
                             <watch-video [cssClass]="'text-primary'" [moduleName]="'DAYBOOK'"></watch-video>
                         </div>
                         <div class="form-group mb-0 cursor-pointer">
-                            <button
+                            <button mat-stroked-button
```

Comment:
```suggestion
                            <button
```

see below this already written 

mat-raised-button
                                color="primary"

Reply:
Done